Dockerfile

```
@@ -1,5 +1,7 @@
 FROM pmariglia/gambit-ubuntu-docker
 
+RUN apt-get clean
+RUN apt-get update
 RUN apt-get install -y python3.6 python3-pip
```

Thanks for opening this.

Looks like a poorly written Dockerfile. I hadn't noticed it because of my cache but I was able to reproduce.

```suggestion
RUN apt-get update && apt-get install -y python3.6 python3-pip
```
From [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). See the section on using RUN with apt-get

---

Haha gotcha, np! I've been playing around with your bot man, its really hype. Can't wait to keep messing around with it!